commands/locations/locations_info.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package locations
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/ystia/yorc/v4/commands/httputil"
+	"github.com/ystia/yorc/v4/helper/tabutil"
+)
+
+func init() {
+	var infoCmd = &cobra.Command{
+		Use:   "info <locationName>",
+		Short: "Gets a location definition",
+		Long:  `Gets the type and the properties of a location.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := httputil.GetClient(ClientConfig)
+			if err != nil {
+				httputil.ErrExit(err)
+			}
+			return getLocationInfo(client, args)
+		},
+	}
+	LocationsCmd.AddCommand(infoCmd)
+}
+
+func getLocationInfo(client httputil.HTTPClient, args []string) error {
```

Comment:
Attempting to get a location info on a bootstrapped secured setup (credentials are stored in a vault, and location property values reference the secret in the vault) shows these warnings for example for an openstack location:
```
2019/11/22 15:27:27 [Warning] template parsing failed for key: "password" value: "{{ secret \"/secret/yorc/locations/myLocation\" \"data=password\" | print }}": template: GenericConfigMap:1: function "secret" not defined
2019/11/22 15:27:27 [Warning] template parsing failed for key: "tenant_name" value: "{{ secret \"/secret/yorc/locations/myLocation\" \"data=tenant_name\" | print }}": template: GenericConfigMap:1: function "secret" not defined
2019/11/22 15:27:27 [Warning] template parsing failed for key: "user_name" value: "{{ secret \"/secret/yorc/locations/myLocation\" \"data=user_name\" | print }}": template: GenericConfigMap:1: function "secret" not defined
```

Reply:
A solution to be discussed to avoid these warnings, could be to add a new function allowing to disable a template resolver, and the config tempalte resolver would be disabled here so it wouldn't try to resolve secret values.
Code diff of the proposal: [diffyorc.zip](https://github.com/ystia/yorc/files/3882697/diffyorc.zip)